crates/nu-cli/tests/support/completions_helpers.rs

```diff
@@ -0,0 +1,105 @@
+use std::path::PathBuf;
+
+use nu_command::create_default_context;
+use nu_engine::eval_block;
+use nu_parser::parse;
+use nu_protocol::{
+    engine::{EngineState, Stack, StateDelta, StateWorkingSet},
+    PipelineData, Span, Value,
+};
+use nu_test_support::fs;
+use reedline::Suggestion;
+const SEP: char = std::path::MAIN_SEPARATOR;
+
+#[allow(dead_code)]
```

Comment:
If I remove those `#[allow(dead_code)]` compiler shows warning, even though it's being used.